external/api/src/main/java/org/codice/compliance/saml/plugin/IdpResponder.java

```diff
@@ -34,10 +43,16 @@
    * where {@code exampleUrl} is the url in the "Location" header returned by the IdP
    * </pre>
    */
-  IdpRedirectResponse getIdpRedirectResponse(Response originalResponse);
+  // TODO When DDF is fixed to return a POST SSO response, change the return type to
+  // `IdpPostResponse`
+  IdpResponse getSSORedirectResponse(Response originalResponse);
```

Comment:
Changed these methods because having `Idp` in the name is redundant and added `SSO` so this interface could be expanded for other profiles.